mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(netPrice) {
+  return netPrice * 1.2;
```

Comment:
I am not sure if netprice is exactly correct to pass in (as I think netprice is after tax has been removed, and this is still before) but I like how specific and readable this label is.